lib/board.rb

```diff
@@ -1,59 +1,54 @@
 class Board
-  attr_reader :board, :tokens
-
   def initialize
     @board = Array.new(9)
     @tokens = {X: "X", O: "O"}
   end
 
-  def place_token(pos)
-    board[pos - 1] = next_token
+  def place_token(position)
+    board[position - 1] = next_token
   end
 
-  def get(pos)
-    board[pos - 1]
+  def get(position)
+    board[position - 1]
   end
 
-  def is_available?(pos)
-    board[pos - 1].nil?
+  def is_available?(position)
+    board[position - 1].nil?
   end
 
-  def game_over?
-    win? || tie?
+  def outcome
+    winner || draw?
```

Comment:
This is a bit of a rehash of a previous comment, but what are the possible return types here, and what meaning is being assigned to them?

Reply:
Will adjust.